subfun/design/setSpeedTargets.m

```diff
@@ -0,0 +1,19 @@
+% [ W I P ]
+
+function speeds = setSpeedTargets(cfg)
+    % Compute the matrix with the speed targets if requested, otherwise output will be only zeros
+    %
+    % (C) Copyright 2021 CPP visual motion localizer developpers
+
+    if ismember('speed', cfg.target.type)
+        [~, nbEventsPerBlock, ~, nbBlocks] = getDesignInput(cfg);
+        speeds = ones(nbBlocks, nbEventsPerBlock) * cfg.dot.speedPixPerFrame;
+
+        % Outputs an "empty" matrix in case no speed task is required
+    else
+        [~, nbEventsPerBlock, ~, nbBlocks] = getDesignInput(cfg);
+        speeds = zeros(nbBlocks, nbEventsPerBlock) * cfg.dot.speedPixPerFrame;
```

Comment:
Have not tested it yet but I think that having zeros might results in dots not moving, will update once tested

Reply:
I confirm that zeros make the dots static

Reply:
```suggestion
        speeds = ones(nbBlocks, nbEventsPerBlock) * cfg.dot.speedPixPerFrame;
```